packages/common/random-access-storage/src/browser/web-fs.ts

```diff
@@ -79,14 +79,19 @@ export class WebFS implements Storage {
   private async _delete(path: string): Promise<void> {
     await Promise.all(
       Array.from(this._getFiles(path)).map(async ([path, file]) => {
-        await file.destroy().catch((err: any) => log.catch(err));
+        await file.destroy().catch((err: any) => log.warn(err));
         this._files.delete(path);
       })
     );
   }
 
   async reset() {
-    await this._delete('');
+    await this._initialize();
+    for await (const filename of await (this._root as any).keys()) {
+      assert(typeof filename === 'string');
```

Comment:
in practice is this ever not a string? currently if this throws the reset will stop I think, may want to just do a conditional check or try/catch?

Reply:
Make sense. It is always a string, but as you saw types are missing, so the `filename` type is `unknown`